test/numbers.coffee

```diff
@@ -13,9 +13,26 @@
 # Binary Integer Literals
 # Binary notation is understood as would be decimal notation.
 
+toJS = (str) ->
+  CoffeeScript.compile str, bare: yes
+  .replace /^\s+|\s+$/g, '' # Trim leading/trailing whitespace
+
```

Comment:
Yeah, that’s been the rule. Only check that the CoffeeScript does what is expected.